bin/build_docker_images.sh

```diff
@@ -259,6 +259,7 @@ build_images () {
     create_image tf_cifar_container TensorFlowCifarDockerfile $public
     create_image tf-container TensorFlowDockerfile $public
     create_image pytorch-container PyTorchContainerDockerfile $public
+    create_image caffe2-container Caffe2Dockerfile $public
```

Comment:
Let's call this `caffe2-onnx-container` and rename the Dockerfile to `Caffe2OnnxDockerfile`